### ISSUE TITLE
Support for "propertyColumns" in azurerm_stream_analytics_output_eventhub 

### ISSUE THREAD

**Issue by oliverchristen** [hashicorp/terraform-provider-azurerm#10805]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

In Azure Portal and in Azure CLI it is supported to configure *propertyColumns*. It should be in Terraform as well. When I manually set it in the Poral, this leads to a broken Terraform config when refreshing state: 
`Error: converting Output Data Source to a EventHub Output: <nil>`

In the Portal it looks like this:
![image](https://user-images.githubusercontent.com/12251419/109787674-c3b90100-7c0e-11eb-8be8-49889113d324.png)

CLI documentation: https://docs.microsoft.com/en-us/cli/azure/ext/stream-analytics/stream-analytics/output?view=azure-cli-latest#ext_stream_analytics_az_stream_analytics_output_create

### New or Affected Resource(s)

* azurerm_stream_analytics_output_eventhub 

### Potential Terraform Configuration

```hcl
resource "azurerm_stream_analytics_output_eventhub" "eventhuboutput" {
  name                      = "eventhuboutput"
  stream_analytics_job_name = azurerm_stream_analytics_job.asa-iothubtoeventhub.name
  resource_group_name       = "rg"
  eventhub_name             = "eventhub"
  servicebus_namespace      = "eventhub-namespace"
  shared_access_policy_key  = "default_primary_key"
  shared_access_policy_name = "RootManageSharedAccessKey"

  property_columns = [
    "Table",
    "Id",
  ]

  serialization {
    type     = "Json"
    encoding = "UTF8"
    format   = "LineSeparated"
  }
}
```

### References

* #0000

**Comment by alex-mozejko** [comment]
heya @katbyte , 

[the api](https://docs.microsoft.com/en-us/rest/api/streamanalytics/2017-04-01-preview/outputs/create-or-replace#eventhuboutputdatasource) which adds support for property columns is a preview, but 4 years old. 

if i add support using this api will it likely be added or are preview api's generally not used?

**Comment by stephybun** [comment]
This was closed by #12947 and released in v2.73.0.
